2022-04-25 11:27:48.532 |-INFO  [main] AnlyTest [55] -| Starting AnlyTest on anlythreedeMacBook-Pro.local with PID 1298 (started by anlythree in /Users/anlythree/my/bus-arrive)
2022-04-25 11:27:48.534 |-INFO  [main] AnlyTest [655] -| The following profiles are active: prod
2022-04-25 11:27:50.100 |-INFO  [main] org.springframework.data.repository.config.RepositoryConfigurationDelegate [249] -| Multiple Spring Data modules found, entering strict repository configuration mode!
2022-04-25 11:27:50.102 |-INFO  [main] org.springframework.data.repository.config.RepositoryConfigurationDelegate [127] -| Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2022-04-25 11:27:50.170 |-INFO  [main] org.springframework.data.repository.config.RepositoryConfigurationDelegate [187] -| Finished Spring Data repository scanning in 52ms. Found 0 Redis repository interfaces.
2022-04-25 11:27:51.746 |-INFO  [main] org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor [181] -| Initializing ExecutorService 'threadPool'
2022-04-25 11:27:51.754 |-INFO  [main] org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor [496] -| Autowired annotation is not supported on static methods: public static void top.anlythree.utils.UrlUtil.setKey(java.lang.String)
2022-04-25 11:27:53.689 |-WARN  [main] org.springframework.boot.autoconfigure.thymeleaf.ThymeleafAutoConfiguration$DefaultTemplateResolverConfiguration [106] -| Cannot find template location: classpath:/templates/ (please add some templates or check your Thymeleaf configuration)
2022-04-25 11:27:55.628 |-INFO  [main] AnlyTest [61] -| Started AnlyTest in 7.682 seconds (JVM running for 9.111)
2022-04-25 11:27:57.376 |-INFO  [main] org.springframework.boot.autoconfigure.logging.ConditionEvaluationReportLoggingListener [136] -| 

Error starting ApplicationContext. To display the conditions report re-run your application with 'debug' enabled.
2022-04-25 11:27:57.386 |-ERROR [main] org.springframework.boot.SpringApplication [837] -| Application run failed
java.lang.IllegalStateException: Failed to execute ApplicationRunner
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:789)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:776)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:322)
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:120)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124)
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:123)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:244)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:227)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: java.lang.NullPointerException: null
	at java.util.Objects.requireNonNull(Objects.java:203)
	at top.anlythree.cache.ACache.addXiaoYuanCity(ACache.java:92)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at top.anlythree.task.RememberCityAndRouteInfoTask.cacheXiaoYuanCityInfo(RememberCityAndRouteInfoTask.java:85)
	at top.anlythree.task.RememberCityAndRouteInfoTask.run(RememberCityAndRouteInfoTask.java:55)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:786)
	... 30 common frames omitted
2022-04-25 11:27:57.415 |-INFO  [main] org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor [218] -| Shutting down ExecutorService 'threadPool'
2022-04-25 11:27:57.567 |-ERROR [main] org.springframework.test.context.TestContextManager [248] -| Caught exception while allowing TestExecutionListener [org.springframework.test.context.web.ServletTestExecutionListener@38aa816f] to prepare test instance [AnlyTest@447543ee]
java.lang.IllegalStateException: Failed to load ApplicationContext
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:132)
	at org.springframework.test.context.support.DefaultTestContext.getApplicationContext(DefaultTestContext.java:123)
	at org.springframework.test.context.web.ServletTestExecutionListener.setUpRequestContextIfNecessary(ServletTestExecutionListener.java:190)
	at org.springframework.test.context.web.ServletTestExecutionListener.prepareTestInstance(ServletTestExecutionListener.java:132)
	at org.springframework.test.context.TestContextManager.prepareTestInstance(TestContextManager.java:244)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.createTest(SpringJUnit4ClassRunner.java:227)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner$1.runReflectiveCall(SpringJUnit4ClassRunner.java:289)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.methodBlock(SpringJUnit4ClassRunner.java:291)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:246)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:97)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:190)
	at org.junit.runner.JUnitCore.run(JUnitCore.java:137)
	at com.intellij.junit4.JUnit4IdeaTestRunner.startRunnerWithArgs(JUnit4IdeaTestRunner.java:69)
	at com.intellij.rt.junit.IdeaTestRunner$Repeater.startRunnerWithArgs(IdeaTestRunner.java:33)
	at com.intellij.rt.junit.JUnitStarter.prepareStreamsAndStart(JUnitStarter.java:235)
	at com.intellij.rt.junit.JUnitStarter.main(JUnitStarter.java:54)
Caused by: java.lang.IllegalStateException: Failed to execute ApplicationRunner
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:789)
	at org.springframework.boot.SpringApplication.callRunners(SpringApplication.java:776)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:322)
	at org.springframework.boot.test.context.SpringBootContextLoader.loadContext(SpringBootContextLoader.java:120)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContextInternal(DefaultCacheAwareContextLoaderDelegate.java:99)
	at org.springframework.test.context.cache.DefaultCacheAwareContextLoaderDelegate.loadContext(DefaultCacheAwareContextLoaderDelegate.java:124)
	... 25 common frames omitted
Caused by: java.lang.NullPointerException: null
	at java.util.Objects.requireNonNull(Objects.java:203)
	at top.anlythree.cache.ACache.addXiaoYuanCity(ACache.java:92)
	at java.util.ArrayList.forEach(ArrayList.java:1259)
	at top.anlythree.task.RememberCityAndRouteInfoTask.cacheXiaoYuanCityInfo(RememberCityAndRouteInfoTask.java:85)
	at top.anlythree.task.RememberCityAndRouteInfoTask.run(RememberCityAndRouteInfoTask.java:55)
	at org.springframework.boot.SpringApplication.callRunner(SpringApplication.java:786)
	... 30 common frames omitted
2022-04-25 11:28:24.607 |-INFO  [main] AnlyTest [55] -| Starting AnlyTest on anlythreedeMacBook-Pro.local with PID 1321 (started by anlythree in /Users/anlythree/my/bus-arrive)
2022-04-25 11:28:24.609 |-INFO  [main] AnlyTest [655] -| The following profiles are active: prod
2022-04-25 11:28:26.359 |-INFO  [main] org.springframework.data.repository.config.RepositoryConfigurationDelegate [249] -| Multiple Spring Data modules found, entering strict repository configuration mode!
2022-04-25 11:28:26.361 |-INFO  [main] org.springframework.data.repository.config.RepositoryConfigurationDelegate [127] -| Bootstrapping Spring Data Redis repositories in DEFAULT mode.
2022-04-25 11:28:26.428 |-INFO  [main] org.springframework.data.repository.config.RepositoryConfigurationDelegate [187] -| Finished Spring Data repository scanning in 54ms. Found 0 Redis repository interfaces.
2022-04-25 11:28:27.939 |-INFO  [main] org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor [181] -| Initializing ExecutorService 'threadPool'
2022-04-25 11:28:27.946 |-INFO  [main] org.springframework.beans.factory.annotation.AutowiredAnnotationBeanPostProcessor [496] -| Autowired annotation is not supported on static methods: public static void top.anlythree.utils.UrlUtil.setKey(java.lang.String)
2022-04-25 11:28:29.788 |-WARN  [main] org.springframework.boot.autoconfigure.thymeleaf.ThymeleafAutoConfiguration$DefaultTemplateResolverConfiguration [106] -| Cannot find template location: classpath:/templates/ (please add some templates or check your Thymeleaf configuration)
2022-04-25 11:28:31.430 |-INFO  [main] AnlyTest [61] -| Started AnlyTest in 7.393 seconds (JVM running for 8.63)
2022-04-25 11:28:31.726 |-INFO  [SpringContextShutdownHook] org.springframework.scheduling.concurrent.ThreadPoolTaskExecutor [218] -| Shutting down ExecutorService 'threadPool'
